composer.json

```diff
@@ -22,6 +22,13 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
+    "support": {
+        "email": "doctrine-user@googlegroups.com",
+        "issues": "https://github.com/doctrine/doctrine2/issues",
+        "source": "https://github.com/doctrine/doctrine2",
+        "rss": "https://github.com/doctrine/doctrine2/releases.atom",
```

Comment:
I wasn't sure what to put here, @Majkl578 what were you thinking about? Something else maybe?

Reply:
Should be fine.